app/components/SystemMetric.tsx

```diff
@@ -93,25 +91,17 @@ export function SiloMetric({
   // in the tooltip. could be just once on the end of the x-axis like GCP
 
   return (
-    <div>
-      <h2 className="flex items-center gap-1.5 px-3 text-mono-sm text-default">
-        {title} {unit && <span className="text-tertiary">({unit})</span>}{' '}
-        {(inRange.isPending || beforeStart.isPending) && <Spinner />}
```

Comment:
The loading state is handled inside the chart. The logic needs to be improved, but it does show up for these charts.